test/unit/image/pixels.js

```diff
@@ -42,6 +42,39 @@ suite('pixels', function() {
     test('works with integers', function() {
       assert.deepEqual(img.get(25, 25), [255, 0, 0, 255]);
       assert.deepEqual(img.get(25, 26), [0, 0, 255, 255]);
+      assert.deepEqual(img.get(0, 0), [255, 0, 0, 255]);
+      assert.deepEqual(img.get(49, 49), [0, 0, 255, 255]);
+    });
+
+    test('works when out of bounds', function() {
+      assert.deepEqual(img.get(25, -1), [0, 0, 0, 0]);
+      assert.deepEqual(img.get(-1, 26), [0, 0, 0, 0]);
+      assert.deepEqual(img.get(25, 50), [0, 0, 0, 0]);
+      assert.deepEqual(img.get(50, 26), [0, 0, 0, 0]);
+    });
+
+    test('get() works', function() {
+      var copy = img.get();
+      assert.instanceOf(copy, p5.Image);
+      assert.equal(copy.width, img.width);
+      assert.equal(copy.height, img.height);
+
+      assert.deepEqual(copy.get(25, 25), [255, 0, 0, 255]);
```

Comment:
can these `assert`s be in their own descriptive `it` block since they test different arguments format and resulting behavior?

Reply:
can you expand on this a bit? i'm not sure what you mean by a 'descriptive it block'. also, these asserts aren't really testing the `get(x,y)` functionality itself (that's testing in a previous test), they're just testing that the pixels in the `copy` are the correct values - it just happens that `get(x,y)` is the clearest way to extract a pixel from an image ;-)

Reply:
>  also, these asserts aren't really testing the get(x,y)

ah ok! that's the context i was missing. i'm good with this test as-is.